cohort_4/playbook/ACTIVITIES.md

```diff
@@ -689,6 +689,35 @@ Uniqueness is also important - no two people should be paired together again
 unless everyone has already been paired with each other. This will help members
 of the club all form bonds with each other.
 
+## Types of Pairing
+
+### Pair programming between beginner-advanced pairs
+
+Pair programming between beginner and advanced hackers is fairly similar to
+normal pair programming. They should still be rotating regularly. The advanced
+hacker will have a tendnacy to drive the coding completely and club leader may
+have to help them resist that urge.
+
+When pairing between beginner and advanced hackers, make sure each hacker
+understands what they are getting out of it:
+
+For the advanced hacker:
+
+- True mastery of a subject can only come once someone is at the point they can
+  explain it to someone. If a hacker can't navigate while pairing, they dont
+  have an in-depth understanding of the topic.
+- This is practice in working with people who don't know as much
+  subject-specific information. Teaching this content to a beginning hacker is
+  practice for any future interaction with teammembers.
+- A team is only as strong as its weakest link. Helping teammembers gain more
+  experience and increase their knowledge can pay off later.
+
+For the beginning hacker:
+
+- Focusing only on the code is a good way to get familiar with syntax and build
+  up muscle memory.
+- Anytime the hacker is lost, they can reach out to the navigator.
```

Comment:
I think it does. It doesn't matter who is at the computer, they should be able to ask each-other questions.